Sources/Crypto/RSAUtils.swift

```diff
@@ -15,3 +16,9 @@ public class RSAUtils {
     return regExp.stringByReplacingMatches(in: key, options: [], range: fullRange, withTemplate: "")
   }
 }
+
+extension String {
+  public var sha256: String? {
+    NYPLStringAdditions.sha256forString(self)
```

Comment:
Just curious, did you intentionally make the `sha` lowercase because it's in Swift? since it's uppercase in ObjC

Reply:
no, that's swift name resolution hard at work :)